rb/lib/selenium/webdriver/remote/capabilities.rb

```diff
@@ -83,38 +83,46 @@ def platform=(value)
 
         class << self
           def chrome(opts = {})
+            WebDriver.logger.deprecate('Remote::Capabilities.chrome', 'Options.chrome', id: :caps_browsers)
             new({
               browser_name: 'chrome'
             }.merge(opts))
           end
 
           def edge(opts = {})
+            WebDriver.logger.deprecate('Remote::Capabilities.edge', 'Options.edge', id: :caps_browsers)
             new({
               browser_name: 'MicrosoftEdge'
             }.merge(opts))
           end
           alias microsoftedge edge
 
           def firefox(opts = {})
+            WebDriver.logger.deprecate('Remote::Capabilities.firefox', 'Options.firefox', id: :caps_browsers)
             new({
               browser_name: 'firefox'
             }.merge(opts))
           end
           alias ff firefox
 
           def safari(opts = {})
+            WebDriver.logger.deprecate('Remote::Capabilities.safari', 'Options.safari', id: :caps_browsers)
```

Comment:
Looks like we have a loading bug in `Options::Safari` caching the first value of browser name:

```ruby
[1] pry(main)> Selenium::WebDriver::Options.safari
=> #<Selenium::WebDriver::Safari::Options:0x0000000109d8b3a0 @options={:browser_name=>"safari"}>
[2] pry(main)> Selenium::WebDriver::Safari.technology_preview!
=> true
[3] pry(main)> Selenium::WebDriver::Options.safari
=> #<Selenium::WebDriver::Safari::Options:0x0000000109d3a6a8 @options={:browser_name=>"safari"}>
```

It might be minor but it doesn't happen in `Capabilities.safari`.

Reply:
Yeah, because of that, in `Options::Safari#as_json` I do this:
```
@options[:browser_name] = Safari.technology_preview? ? 'Safari Technology Preview' : 'safari'
```

Reply:
We changed browser name to use `BROWSER_NAME` constant at class load, which means it won't change when `#technology_preview!` is set. :-/ 
Found it when running tests over break, and fixed here: https://github.com/SeleniumHQ/selenium/commit/04d571231191e516480ae8acd1340878210e1861